apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/V1/SessionPoolTests.StatisticsTests.cs

```diff
@@ -58,14 +59,26 @@ public void AllProperties()
                     readWriteTransactionRequestsPrewarmed: 500L,
                     healthy: true,
                     shutdown: false);
-                var allStats = new[] { db1Stats, db2Stats, db3Stats };
+                var db4Stats = new DatabaseStatistics(
+                    new SessionPoolKey(
+                    new DatabaseName("project", "instance", "db1"), "dbrole"),
+                    activeSessionCount: 5,
+                    readPoolCount: 0,
+                    readWritePoolCount: 0,
+                    inFlightCreationCount: 4,
+                    pendingAcquisitionCount: 0,
+                    readWriteTransactionRequests: 100L,
+                    readWriteTransactionRequestsPrewarmed: 95L,
+                    healthy: true,
+                    shutdown: false);
+                var allStats = new[] { db1Stats, db2Stats, db3Stats, db4Stats };
                 var statistics = new Statistics(allStats);
 
                 Assert.Equal(allStats, statistics.PerDatabaseStatistics);
```

Comment:
I just realized, we need to deprecate `SessionPools.Statistics.PerDatabaseStatistics` and add `SessionsPools.Statistics.PerSessionPoolKeyStatistics`.

Implementation wouldn't change, is the name of the property that is wrong now.